utopia-api/src/pragma/pragma.tsx

```diff
@@ -119,10 +119,15 @@ export const jsx = (type: any, ...pragmaParams: any[]) => {
             ...childStyleProps,
             ...childStylesThatOverwriteStyle[index],
           }
-          return React.cloneElement(child as any, {
-            ...removeLayoutPropFromReactBuiltins,
-            style: layoutEnhancedStyleProp,
-          })
+
+          const updatedProps =
+            (child.type as any)?.theOriginalType?.toString() === 'Symbol(react.fragment)'
```

Comment:
in the monkey code file, I used `type == React.Fragment` and that seemed to work for me

Reply:
> in the monkey code file, I used `type == React.Fragment` and that seemed to work for me

I wrote that first, but it didn't work! The type is React.Element here, only the `theOriginalType` shows that it's a fragment.

Reply:
and `theOriginalType == React.Fragment` does not work?